src/odin/directionsbuilder.cc

```diff
@@ -21,46 +21,6 @@ constexpr auto kMinEdgeLength = 0.001f;
 namespace valhalla {
 namespace odin {
 
-const std::unordered_map<int, DirectionsLeg_VehicleType> translate_vehicle_type{
```

Comment:
these arent needed because now directions and trip share the same definition of these things